homeassistant_satellite/__main__.py

```diff
@@ -69,6 +69,7 @@ async def main() -> None:
         choices=("http", "https"),
         help="Home Assistant protocol",
     )
+    parser.add_argument("--skipVerify", action="store_true", help="Skip verification of the SSL certificate")
```

Comment:
```suggestion
    parser.add_argument("--skip-certificate-verification", action="store_true", help="Skip verification of the SSL certificate")
```

Existing arguments use lowercase and dashes in between words, so you should follow that. Even then `skip-verify` would be too generic IMO.

Reply:
made it --skip-verify. you you really want it to extend ? There is not really another thing to confuse it with, apart from that it has a help text